x-pack/plugins/snapshot_restore/common/types/snapshot.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface Snapshot {
+  id: string;
+  summary: SnapshotSummary;
+  repositories: string[];
+}
+
+export interface SnapshotSummary {
+  status: string;
+  start_epoch: string; // e.g. '1554501400'. This and other numerical values are typed as strings.
```

Comment:
With Typescript we can get the helful doc whenever using the interface. But... it has to be written like this:

```ts
export interface SnapshotSummary {
  status: string;
  /** This and other numerical values are typed as strings. e.g. '1554501400'.  */
  start_epoch: string;
  /** e.g. '21:56:40' */
  start_time: string;
  end_epoch: string;
  /** e.g. '21:56:45' */
  end_time: string;
  /** Includes unit, e.g. '4.7s' */
  duration: string;
  indices: string;
  successful_shards: string;
  failed_shards: string;
  total_shards: string;
}
```

<img width="755" alt="Screen Shot 2019-04-08 at 10 03 55" src="https://user-images.githubusercontent.com/2854616/55708228-ac067d00-59e5-11e9-9ff3-8430edcf4496.png">